src/agenteval/leaderboard/models.py

```diff
@@ -47,7 +47,7 @@ def __str__(self):
             }
             configs.append(config_def)
         yaml_section = yaml.dump({"configs": configs})
-        return f"---\n{yaml_section.strip()}\n---{self.text_content.lstrip()}"
+        return f"---\n{yaml_section.strip()}\n---\n{self.text_content.lstrip()}"
```

Comment:
Without this I got
> No YAML front matter found in README.md

the next time I ran something that tried to do the readme check, after making my initial change to add a new config.

https://huggingface.co/datasets/allenai/asta-bench-internal-results/commit/ada319c806dcce3718042abf34a07bf0891b35b8